data/hai/hai.txt

```diff
@@ -419,11 +440,12 @@ outfitter "Hai Advanced"
 	"Cargo Scanner"
 	"Sand Cell"
 
-shipyard "Unfettered Basics"
+shipyard "Unfettered"
```

Comment:
It's a breaking change to rename these shipyards and outfitters.

Reply:
oh sorry, was not aware of that, because the map is part of the data folder and reference to said shipyard is only in the data folder, not in the save files, I'll revert that change then I guess

but still it feels pretty weird to have them named Basics and give them Shield Beetles x)

Reply:
It's a game object that can be extended by the event system. It is also available for use by any plugin (just like any plugin can use an existing ship or outfit in it's content). Renaming them means any plugin would be referring to a now-missing list of ships or outfits.

W.r.t. adding things to it, if an item doesn't make thematic sense, then introduce a new outfitter where the item will make thematic sense.